app/controllers/api/admin/commodities/search_references_controller.rb

```diff
@@ -19,8 +19,8 @@ def collection_url
         def commodity
           @commodity ||= begin
             commodity = Commodity.actual
-                   .declarable
```

Comment:
Replaced by the scope filter `by_productline_suffix`. This just restricted us to creating references non-declarable and declarable commodities that had a productline suffix of '80' . We actually want to be able to create references for both declarable and non-declarable commodities.